orchagent/portsorch.cpp

```diff
@@ -3918,3 +3918,29 @@ void PortsOrch::getPortSerdesVal(const std::string& val_str,
         lane_values.push_back(lane_val);
     }
 }
+
+void PortsOrch::increasePortNeighRefCount(const string &alias)
```

Comment:
I am assuming that you don't need to know the name/OID of the neighbor entry that is depending on port. That would really help in debugging scale issues! Think about it!